Formula/sox.rb

```diff
@@ -3,6 +3,7 @@ class Sox < Formula
   homepage "https://sox.sourceforge.io/"
   url "https://downloads.sourceforge.net/project/sox/sox/14.4.2/sox-14.4.2.tar.gz"
   sha256 "b45f598643ffbd8e363ff24d61166ccec4836fea6d3888881b8df53e3bb55f6c"
+  license "GPL-2.0-only"
```

Comment:
What did you base this on? https://sourceforge.net/p/sox/code/ci/master/tree/src/downsample.c for example seems to disagree

Reply:
Well https://sourceforge.net/projects/sox/ under the heading of "License" says so. 

Interesting to note, that Wikipedia https://en.wikipedia.org/wiki/SoX#cite_note-license-1 speaks of GPL-2.0-or-later and refers to https://sourceforge.net/p/sox/code/ci/master/tree/COPYING as the source; this is a file from sourceforge, but Wikipedia comes to a different conclusion, than explicitly stated at sourceforge (GPL-2.0-only as I have cited at the top of my answer). 

On the other hand, you are totally right when citing example source-files with GPL-2.1-or-later, but this does apply to 147 of 241 .c and .h files only, so I think it cannot be generalized.

Reply:
Just added another commit, which is comment-only and gives the source of the license term.
So in the future, when sox changes to a another license (hopefully GPL-2.1-or-later), the formula can be easily adjusted.

Reply:
I think it should be defined as both `GPL` and `LGPL`.

Reply:
Yes, I agree; thanx for pointing this out ! Added a commit to reflect this.